src/AccessibilityInsights.ExtensionsTests/ContainerUnitTests.cs

```diff
@@ -3,49 +3,95 @@
 using AccessibilityInsights.Extensions;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using System.Collections.Generic;
-using System.IO;
 using System.Linq;
-#if FAKES_SUPPORTED
-using Microsoft.QualityTools.Testing.Fakes;
-using System.IO.Fakes;
-#endif
+using AccessibilityInsights.Extensions.Interfaces.IssueReporting;
+using AccessibilityInsights.ExtensionsTests.DummyClasses;
+using AccessibilityInsights.Extensions.Interfaces.Upgrades;
+using AccessibilityInsights.Extensions.Interfaces.Telemetry;
+using System;
+using System.Reflection;
+using System.IO;
 
 namespace AccessibilityInsights.ExtensionsTests
 {
     [TestClass]
     public class ContainerUnitTests
     {
+        const string NoMatchOverride = "this_file_does_not_exist.dll";
```

Comment:
These two variable names aren't so informative. How about something indicating that one path is valid and one isn't?